browser/tor/brave_tor_browsertest.cc

```diff
@@ -152,12 +152,10 @@ class BraveTorTest : public InProcessBrowserTest {
 
   BraveTorTest() {
     // Disabling CSP on webui pages so EvalJS could be run in main world.
```

Comment:
leftover comment? Also in `browser/ui/webui/brave_wallet/wallet_panel_ui_browsertest.cc`, `browser/ui/webui/settings/brave_extensions_manifest_v2_browsertest.cc` and `browser/ui/webui/settings/brave_tor_snowflake_extension_browsertest.cc`

Reply:
Thanks @darkdh, fixed in https://github.com/brave/brave-core/pull/22320/commits/392096b2e4736035853b803866a581d95c2be4f5